package/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -x
+
+if [ "$1" = "" ]; then
+    bin/${PROG} agent $@
+else
+    bin/${PROG} $@
+fi
```

Comment:
Still need to make sure that k3s is pid 1 or this blows up on cgroupv2 with:
```console
E0816 21:13:20.085475       7 node_container_manager_linux.go:61] "Failed to create cgroup" err="cannot enter cgroupv2 \"/sys/fs/cgroup/kubepods\" with domain controllers -- it is in an invalid state" cgroupName=[kubepods]
W0816 21:13:20.085480       7 watcher.go:93] Error while processing event ("/sys/fs/cgroup/kubepods": 0x40000100 == IN_CREATE|IN_ISDIR): open /sys/fs/cgroup/kubepods: no such file or directory
E0816 21:13:20.085498       7 kubelet.go:1378] "Failed to start ContainerManager" err="cannot enter cgroupv2 \"/sys/fs/cgroup/kubepods\" with domain controllers -- it is in an invalid state"
```

```suggestion
    exec bin/${PROG} agent $@
else
    exec bin/${PROG} $@
fi
```